lib/rest-adapter.js

```diff
@@ -571,16 +577,16 @@ RestAdapter.prototype.allRoutes = function() {
 };
 
 RestAdapter.prototype.getClasses = function() {
-  return this.remotes.classes(this.options).map(function(c) {
-    return new RestClass(c);
+  return this.remotes.classes(this.options).map(c => {
+    return new RestClass(c, this.options);
```

Comment:
Is it OK to use ES6 style in the code? I thought it was still all written in ES5 manner...